src/elements/Tree.tsx

```diff
@@ -0,0 +1,321 @@
+import * as React from "react";
+import clsx from "clsx";
+import Box, { BoxProps } from "@vertigis/web/ui/Box";
+import Checkbox from "@vertigis/web/ui/Checkbox";
+import ChevronDownIcon from "@vertigis/web/ui/icons/ChevronDown";
+import Collapse from "@vertigis/web/ui/Collapse";
+import DynamicIcon, { DynamicIconProps } from "@vertigis/web/ui/DynamicIcon";
+import { FormElementProps, FormElementRegistration } from "@geocortex/workflow/runtime";
+import IconButton, { IconButtonProps } from "@vertigis/web/ui/IconButton";
+import Link from "@vertigis/web/ui/Link";
+import List, { ListProps } from "@vertigis/web/ui/List";
+import ListItemIcon from "@vertigis/web/ui/ListItemIcon";
+import ListItemText from "@vertigis/web/ui/ListItemText";
+import ListItemButton from "@vertigis/web/ui/ListItemButton";
+import { UIContext } from "@vertigis/web/ui/UIContext";
+import { BrandingService } from "@vertigis/web/branding/BrandingService";
```

Comment:
Referencing something outside of `@vertigis/web/ui` takes us in a direction we'll never be able to come back from. 

Even using `@vertigis/web/ui/UIContext` is an issue. Everything else we reference is a vanilla React component. These other ones are using specific Web functionality that I want to try to avoid avoid.

Reply:
I'm only really using it for the density so I've removed it and set that back as a property.